### PR TITLE
Bug 1862955 - add (optional) session_id & session_count to Glean client info

### PR DESCRIPTION
**Pull request by rosahbruno**
Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [x] If adding a new field, the field should have a description (see #576 for an example)
- [x] If coming from a fork, run integration tests: `./.github/push-to-trigger-integration <username>:<branchname>`

For glean changes:
- [x] Update `templates/include/glean/CHANGELOG.md`

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

**Comment by rosahbruno** [comment]
@akkomar 
Would you be able to review this?

File: templates/include/glean/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Version 1
 
+- Added optional `session_id` and optional `session_count` client info field. See [Bug 1862955](https://bugzilla.mozilla.org/show_bug.cgi?id=1862955).
```
Comment (whd) [inline templates/include/glean/CHANGELOG.md:3]: Side note: looks like  https://bugzilla.mozilla.org/show_bug.cgi?id=1751739 is missing from this history.

**Review by whd — COMMENTED** [review]
> @whd this will add two fields to all Glean tables. Do you want a heads-up before merge?

No unless you want to manually verify the efficacy of this PR in stage without automatically propagating to prod. Scanning history I can't recall being involved in the previous changes to these schemas. I'm mostly interested in PB[ER] changes since those can require an ingestion-sink code update.

File: templates/include/glean/CHANGELOG.md (hunk 1 above)
Comment (rosahbruno) [reply templates/include/glean/CHANGELOG.md:3]: Fixed https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/795/commits/4e2238736edf229b1073d1def879703b6d7a6adf

**Comment by rosahbruno** [comment]
@akkomar 
The Glean.js PRs have landed and I added the missing CHANGELOG entry that @whd pointed out. This should be ready for review again.

**Review by akkomar — APPROVED** [review]
I have pushed this to a branch in this repo to confirm that all tests are passing: https://app.circleci.com/pipelines/github/mozilla-services/mozilla-pipeline-schemas/3369/workflows/67927c70-749e-4dea-b2c0-712240212177

**Pull request merged**